### PR TITLE
Updated mailing addresses per @jameshupp

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated the mailing address titles under the map to match the titles at the top.

## Additions

- None

## Removals

- None

## Changes

- Updated address titles

## Testing

- fetch and navigate to `/contact-us`

## Review

- @anselmbradford 
- @sebworks 
- @KimberlyMunoz 

## Preview

screen shot 2015-04-28 at 4 40 52 pm

## Notes

- The map is for the old office, which is empty. That feels a bit odd, should we update this in another PR?

### CODE REVIEW

**Comment by anselmbradford** [comment]
:+1: 

> The map is for the old office, which is empty. That feels a bit odd, should we update this in another PR?

Sounds like something for another PR.

**Comment by Scotchester** [comment]
If these are mailing addresses, I recommend adhering to the USPS formatting standards. Specifically, they should look like this:

```
Consumer Financial Protection Bureau
1700 G St NW
Washington DC 20552-0003

Consumer Financial Protection Bureau
PO Box 4503
Iowa City IA 52244-4503
```

Sidenote: Love the passive aggressive note on their website when using the ZIP code lookup. "... using standard abbreviations and formatting…"

screen shot 2015-04-27 at 17 48 55

**Comment by KimberlyMunoz** [comment]
I dug up the [GPO style guide](http://www.gpo.gov/fdsys/pkg/GPO-STYLEMANUAL-2008/pdf/GPO-STYLEMANUAL-2008.pdf), because not having a comma between the city and state seems wrong to me.

They suggest:

- Words such as Street, Avenue, Place, Road, Square, Boulevard,
Terrace, Drive, Court, and Building, following a name or number,
are abbreviated in footnotes, sidenotes, tables, leaderwork, and
lists.
- In addresses, a single period is used with the abbreviations NW.,
SW., NE., SE. (indicating sectional divisions of cities) following
name or number. North, South, East, and West are spelled out at all
times.
- The Postal Service style of two-letter State, Province, and freely
associated State abbreviations is to be used.

And have things formatted like this:
image

I don't like that it omits the +4 of the ZIP code and the period after NW seems unnecessary, but I think that it's a better way to go since I can't find another government site that uses the USPS format on their about page.

**Comment by Scotchester** [comment]
It's bizarre to me that GPO has their own standard for this. Why wouldn't
they follow the USPS?

**Comment by jimmynotjim** [comment]
Something to keep in mind, USPS standard was specifically created for machines to process addresses faster, where as GPO or any other's are going to be more for human readability. Since very few people will be copying the address and pasting it to a shipping label (most systems require each individual field rather than grab all), I'd say we should go the human readable approach with the addition of the +4 zip extension.

**Comment by jameshupp** [comment]
I'm confused as to why we would use all of their style except all caps. Seems to me we either use their standard or we use a different standard focused on what makes the most sense for us.

In this case 1) we're trying to guide someone who is writing as much as typing, and 2) we're guiding their addressing, not doing the addressing. So it's not about conformity to technical standard but about clarity for a user who needs to conform to the standard. Odd as it may seem, I don't think these two rationales lead to identical instructions.

One other note, and it's relevant to @KimberlyMunoz's GPO point: the CFPB style guide describes our own address as "1700 G Street N.W.", which is AP-style compliant. But we should draw a line between addresses described within narrative like a paragraph and addresses provided in address blocks. They serve different purposes (for instance, spelling out the state name is clearly wrong here). The style guide doesn't speak to address blocks yet.

### My preference, short version
no punctuation except between city and state, spell out DC streets, use the two-letter state abbreviation, two spaces before zip. But if anyone has a strong argument on the quadrant either way, I'll bite.

### Rationale
1. The USPS says the comma between city and state is acceptable but no comma is preferred. I find the **comma between city and state** to be helpful for readability. The uniformity of block letters is nice, but without that, you've got ascenders and descenders and other messy things. It's nice to have some visual cue to break up the text into constituent pieces and create some space.

2. I also think **two spaces before the zip code** (which is the USPS preferred standard) similarly helps break up the last line visually.

3. The **(state abbreviation should be two letters**, per official USPS standard (Iowa -> IA; D.C. -> DC). Again, we're trying to guide people to create something for sending via USPS.

4. **No periods** in "PO Box" or "NW". I don't see any advantage in breaking from USPS style here.

5. Spell out "street" for single-letter or numbered DC streets. Otherwise it looks as much like some kind of cypher as it does an address. 1700 G ST NW <- Blech!

6. The comma before the quadrant is part of both DC city government's and GPO's style but it is not part of USPS or AP style. I'm indifferent to it, honestly. As @sarahsimpson09 pointed out to me, the comma helps set off the directions as almost a separate field with its own content that matters. But it also seems to provide more breaking-up-text than necessary, given the nature of downtown DC street names:

1700 G Street NW
vs.
1700 G Street, NW

**Comment by schaferjh** [comment]
+1 for human readability. I agree with @jameshupp all around and vote no on the comma before the quadrant and yes on the +4 to the zip code.

**Comment by Scotchester** [comment]
I :heart: you all. I was _partially_ trolling :trollface: but you all took it seriously. I also agree with everything the brilliant @jameshupp wrote, though I feel pretty strongly about omitting the comma before the DC quadrant name. I don't think there's enough precedent (standardized or colloquial) for including that. I feel like the vast majority of DC addresses I see don't use a comma before the quadrant.

**Comment by KimberlyMunoz** [comment]
I agree with @jameshupp, especially with all the reasons he outlined above. I think the +4 will help citizens get their mail to the CFPB faster and most of those recommendations strike a balance between something the USPS will like and something that is human readable.

On a side note, today I learned CFPB has a style guide. I'll have to check that out.

**Comment by jameshupp** [comment]
@KimberlyMunoz Warning: the guide needs lots of work and someday when there is time it will be the most fun project in the world.

**Comment by jimmynotjim** [comment]
Updated to match @jameshupp's recommendations (see the updated screenshot in PR text above). We good here?

File: contact-us/_vars-contact-us.html [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
 %}
 
 {% set consumer_contact_center = {
-	'title'    : 'Contact Center',
+	'title'    : 'Submit a Complaint',
 	'name'     : 'Consumer Financial Protection Bureau',
-	'street'   : 'P.O. Box 4503',
+	'street'   : 'PO Box 4503',
 	'city'     : 'Iowa City',
 	'state'    : 'Iowa',
```
Comment (KimberlyMunoz) [inline contact-us/_vars-contact-us.html]: Change 'Iowa' to 'IA'.

**Comment by KimberlyMunoz** [comment]
:checkered_flag: :+1: :checkered_flag:

**Comment by anselmbradford** [comment]
:+1:

**Pull request merged**